scripts/analysis/gradle_properties/README.md

```diff
@@ -9,7 +9,7 @@ To count gradle properties keys occurrence statistics run:
 ``` 
 python3 scripts/analysis/gradle_properties/gradle_properties_analysis.py 
```

Comment:
Please, use -m argument since we can have troubles with imports with running as a python file (change it everywhere, please)